deploy/olm-catalog/integreatly-operator/2.2.0/integreatly-operator.v2.2.0.clusterserviceversion.yaml

```diff
@@ -365,8 +365,10 @@ spec:
           - apps
           resources:
           - deployments/finalizers
+          - replicasets
```

Comment:
This and the verb addition below will need to be removed, the existing CSVs shouldn't be modified.

Reply:
Sure, they already added in role.yaml and I removed them from the csv.